src/wasm-lib/src/toml.rs

```diff
@@ -1,13 +1,14 @@
 //! Functions for interacting with TOML files.
 //! We do this in rust because the Javascript TOML libraries are actual trash.
 
+use kcl_lib::settings::types::Configuration;
 use wasm_bindgen::prelude::wasm_bindgen;
 
 #[wasm_bindgen]
 pub fn toml_stringify(json: &str) -> Result<String, String> {
     console_error_panic_hook::set_once();
 
-    let value: serde_json::Value = serde_json::from_str(json).map_err(|e| e.to_string())?;
+    let value: Configuration = serde_json::from_str(json).map_err(|e| e.to_string())?;
```

Comment:
i think this will break when you send a ProjectConfiguration

Reply:
i have a better fix for this here i meant to do this awhile ago since us not saving the right version of the settings has been a problem for a long time https://github.com/KittyCAD/modeling-app/pull/5563

Reply:
Merged @jessfraz 's PR and making use of it here

Reply:
Actually all that work's been done by jess; fricken awesome.